### ISSUE TITLE
[Docs] Include a doc for starting kubernetes on laptop with `sky local up`

### ISSUE THREAD

**Issue by Michaelvll** [skypilot-org/skypilot#4477]
It seems we only have a page that starts the kubernetes cluster when we have the IPs, but we should explicitly mention that `sky local up` without IPs is possible and useful: https://docs.skypilot.co/en/latest/reservations/existing-machines.html

_Version & Commit info:_
* `sky -v`: PLEASE_FILL_IN
* `sky -c`: PLEASE_FILL_IN

**Comment by cg505** [comment]
We already have this: https://docs.skypilot.co/en/latest/reference/kubernetes/kubernetes-deployment.html#kubernetes-setup-kind
Do you mean to add a link on the existing-machines page?

**Comment by Michaelvll** [comment]
Yes, either add a link there or embedded in the existing machines page. Otherwise, it is pretty hard to find that `local up` page.

**Comment by gaocegege** [comment]
I hope it could be included in the [getting-started/installation](https://docs.skypilot.co/en/latest/getting-started/installation.html#cloud-account-setup) page as a user. It provides the simplest way to try out SkyPilot and serves as a true quick start guide.
